packages/private-build-infra/src/debug-macros.js

```diff
@@ -23,26 +26,25 @@ module.exports = function debugMacros(app, isProd, compatVersion) {
       {
         flags: [
           {
-            source: '@ember-data/private-build-infra',
-            flags: PACKAGES,
+            source: '@ember-data/private-build-infra/deprecations',
+            flags: DEPRECATIONS,
           },
         ],
       },
-      '@ember-data/optional-packages-stripping',
+      '@ember-data/deprecation-stripping',
     ],
     [
-      debugMacrosPath,
+      ConvertExistenceChecksToMacros,
       {
-        flags: [
-          {
-            source: '@ember-data/private-build-infra/deprecations',
-            flags: DEPRECATIONS,
-          },
-        ],
+        HAS_EMBER_DATA_PACKAGE: 'ember-data',
+        HAS_STORE_PACKAGE: '@ember-data/store',
+        HAS_MODEL_PACKAGE: '@ember-data/model',
+        HAS_RECORD_DATA_PACKAGE: '@ember-data/record-data',
+        HAS_ADAPTER_PACKAGE: '@ember-data/adapter',
+        HAS_SERIALIZER_PACKAGE: '@ember-data/serializer',
+        HAS_DEBUG_PACKAGE: '@ember-data/debug',
```

Comment:
this one requires special casing, let's transform all except it and rely on the old transform for this one